event-processing/src/main/java/com/camunda/consulting/eventprocessing/Mapper.java

```diff
@@ -0,0 +1,183 @@
+package com.camunda.consulting.eventprocessing;
+
+import com.camunda.consulting.eventprocessing.EventController.CreateEventRequest;
+import com.camunda.consulting.eventprocessing.EventController.CreateEventResponse;
+import com.camunda.consulting.eventprocessing.EventController.GetEventResponse;
+import com.camunda.consulting.eventprocessing.EventRepository.EventEntity;
+import com.camunda.consulting.eventprocessing.EventRepository.EventEntity.State;
+import com.camunda.consulting.eventprocessing.EventService.Event;
+import com.camunda.consulting.eventprocessing.EventService.UnsavedEvent;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
```

Comment:
What about using an explicit example here?

This generic reflection stuff is quite difficult to understand.

Reply:
This mapper is just a util to map between Entity - internal pojo - rest dto